packages/core/helper-plugin/lib/src/hooks/useFetchClient/index.js

```diff
@@ -0,0 +1,10 @@
+import { useContext } from 'react';
+import StrapiAppContext from '../../contexts/StrapiAppContext';
+
+const useFetchClient = () => {
```

Comment:
I don't know how often the StrapiAppContext  changes if ever but worth check if you need a memo in there

Reply:
Do you mean using a memo like `useContext(useMemo(StrapiAppContext)` ?

Reply:
I wouldn't do it this way but it doesn't matter if the StrapiAppContext isn't changing anyway you won't need it I was curious if you verified that :)